src/site/stages/build/process-cms-exports/transform.js

```diff
@@ -38,19 +38,27 @@ function getEntityTransformer(entityType, verbose = true) {
  * @param {Object} entity - The contents of the entity itself before
  *                          reference expansion and property
  *                          transformation.
+ * @param {string} uuid - The UUID of the current entity
+ * @param {Array<Object>} ancestors - All the current entity's
+ *                        ancestors. ancestors[1] is the child of
+ *                        ancestors[0], etc.
+ *                        ancestors[ancestors.length] is the current
+ *                        entity's direct parent.
```

Comment:
Would the current entity's direct parent be  `ancestors[ancestors.length - 1]`?

Reply:
:man_facepalming: Yeah, that. :stuck_out_tongue: